src/main/resources/assets/xpbook/lang/pt_br.json

```diff
@@ -0,0 +1,7 @@
+{
+    "item.xpbook.xp_book": "ANTIGO - CLIQUE DIREITO PARA CONVERTER",
+    "item.xpbook.xp_tome": "XP Tome",

```

Comment:
This doesn't seem to be translated.

Reply:
There is no translation or adaptation for this, so the name of the plugin is the same Unless you can explain to me what "Tome" would be

XP in Brazil is the same as the English language, in the Portuguese language of Brazil there are many things that are not translated... mouse... chipset... mobo... monitor are an example

Reply:
A time is "a book, especially a large, heavy, scholarly one".

Reply:
The closest I could find to this would be a "dicionário" (dictionary) in my language

I think it would look weird "Dicionário de XP" (XP Dictionary)

I searched and didn't find other things, maybe I haven't... or if there is, I haven't found it

Reply:
Ok, thank you for checking. I'll merge this as it is then!